src/data/customer/Route.js

```diff
@@ -79,6 +79,97 @@ export const createRouteLocationAsync = async (
   }
 };
 
+
+/** Update driver time slot
+ * @param {int} scheduleId
+ * @param {object} payload {driver_id, transactionGroupId, startTime, endTime, startDate}
```

Comment:
Please change this to **driverId**.
all function arguments and object keys should be in camelCase